package.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Not sure we need this in the root, would be a bit weird for it to work on anything other than CLI. Maybe we should group the other folders under a project?